calling-apis/managing-asynchronous-calls.md

```diff
@@ -0,0 +1,277 @@
+# Managing Asynchronous Calls
+
+## Goal
+
+The goal for this lesson is to witness how asynchronous code can effect our data management.
+
+We should walk away from this lesson with one strategy to manage asynchronous code. We should also be able to question and research what other strategies may exist.
+
+## Experiment: Making Multiple Calls
+
+Raffy is building a map web app. He wants to run an experiment!
+
+He wants to make two API requests to the [LocationIQ APIs](https://locationiq.com/docs):
+
+1. First, he wants to make a request to the Forward Geocoding API to find the latitude and longitude of "Seattle, Washington, USA."
+1. Then, he wants to make a request to the Reverse Geocoding API with the latitude and longitude he found, and see if the results come back as "Seattle, Washington, USA."
+
+He decides to run his experiment with [this code](https://replit.com/@adacore/Managing-Asynchronous-Requests)!
+
```

Comment:
I'm going to suggest modifying the code examples to use secrets rather than hard-coded api keys, so I thought a callout here to remind the reader to set up their secret would be helpful.
```suggestion

### !callout-warning

## Raffy's LocationIQ API Key

Raffy knows that it's important to keep his API key private, so he set up a secret in his REPL to keep the key out of his code. If we want to try running Raffy's code, we'll need to do the same thing by opening the Secrets panel, and adding a new key named `api_key` with our LocationIQ API key as the value.

### !end-callout

```